code/modules/events/spacevine.dm

```diff
@@ -428,8 +428,8 @@
 	if(potency != null)
 		mutativeness = potency / 10
 	if(production != null)
-		spread_cap *= production / 5
-		spread_multiplier /= production / 5
+		spread_cap /= production / 5
+		spread_multiplier *= production / 5

```

Comment:
When I was tweaking Kuduz and repeatedly fixing it, I did half wonder if this was the incorrect formula.